src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,33 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private double side;
+    private double upperBase;
+    private double lowerBase;
+
+    public IsoscelesTrapezoid(double upperBase,double lowerBase, Color color) {
+        super(color);
+        this.upperBase = upperBase;
+        this.lowerBase = lowerBase;
+    }
+
+    @Override
+    public double calculateArea() {
+        return Math.round((upperBase + lowerBase) / 2 * calculateHeight());
+    }
+
+    public double calculateHeight() {
+        return Math.sqrt(Math.pow(side, 2) - Math.pow(((upperBase + lowerBase) / 2), 2));
```

Comment:
The formula for calculating the height of an isosceles trapezoid is incorrect. It should be `sqrt(side^2 - ((lowerBase - upperBase) / 2)^2)`. Please correct this to ensure accurate calculations.